src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/update/OverseasEntityUpdateSubmissionDtoValidatorTest.java

```diff
@@ -0,0 +1,201 @@
+    {
+        beneficialOwnerCorporateDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerCorporateDto());
+    };
+    private final List<BeneficialOwnerGovernmentOrPublicAuthorityDto> beneficialOwnerGovernmentOrPublicAuthorityDtoList = new ArrayList<>();
+    {
+        beneficialOwnerGovernmentOrPublicAuthorityDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerGovernmentOrPublicAuthorityDto());
+    };
+
+    private final List<ManagingOfficerIndividualDto> managingOfficerIndividualDtoList = new ArrayList<>();
+    {
+        managingOfficerIndividualDtoList.add(ManagingOfficerMock.getManagingOfficerIndividualDto());
+    };
+    private final List<ManagingOfficerCorporateDto> managingOfficerCorporateDtoList = new ArrayList<>();
+    {
+        managingOfficerCorporateDtoList.add(ManagingOfficerMock.getManagingOfficerCorporateDto());
+    };
+
+    @Test
+    void testRegistrationSubmissionCalledWhenEntityNumberIsNullAndUpdateFlagTrue() {
+        setIsRoeUpdateEnabledFeatureFlag(true);
+        buildOverseasEntityUpdateSubmissionDtoWithFullDto();
+        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+        verify(entityDtoValidator, times(1)).validate(eq(entityDto), any(), any());
+        verify(presenterDtoValidator, times(1)).validate(eq(presenterDto), any(), any());
+        verify(dueDiligenceDataBlockValidator, times(1)).validateDueDiligenceFields(
+                eq(overseasEntitySubmissionDto.getDueDiligence()),
+                eq(overseasEntitySubmissionDto.getOverseasEntityDueDiligence()),
+                any(),
+                any());
+        verify(ownersAndOfficersDataBlockValidator, times(1)).validateOwnersAndOfficersAgainstStatement(eq(overseasEntitySubmissionDto),any(),any());
+        assertFalse(errors.hasErrors());
+    }
+    @Test
+    void testUnsuccessfulOverseasEntityUpdateSubmissionWhenUpdateFlagOff() {
+        setIsRoeUpdateEnabledFeatureFlag(false);
+        overseasEntitySubmissionDto.setEntityNumber("OE111229");
+        buildOverseasEntityUpdateSubmissionDtoWithFullDto();
+        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+        verify(entityDtoValidator, times(1)).validate(eq(entityDto), any(), any());
+        verify(presenterDtoValidator, times(1)).validate(eq(presenterDto), any(), any());
+        verify(dueDiligenceDataBlockValidator, times(1)).validateDueDiligenceFields(
+                eq(overseasEntitySubmissionDto.getDueDiligence()),
+                eq(overseasEntitySubmissionDto.getOverseasEntityDueDiligence()),
+                any(),
+                any());
+        verify(ownersAndOfficersDataBlockValidator, times(1)).validateOwnersAndOfficersAgainstStatement(eq(overseasEntitySubmissionDto),any(),any());
+        assertFalse(errors.hasErrors());
+    }
+    @Test
```

Comment:
I think these tests are probably duplicating the logic present in the `OverseasEntitySubmissionDtoValidatorTest` test class (copy-and-paste?). 

In theory, we just want to run the same tests again but for the update - not registration - scenario, yes? If so, might just be better bundling these tests into the other test class and getting code reuse that way.